lms/static/sass/views/_login-register.scss

```diff
@@ -503,6 +507,12 @@ $sm-btn-linkedin: #0077b5;
         }
     }
 
+    .institution-list {
+        li {
+            line-height: 1.5em;
```

Comment:
Please refrain from using ems for units of measurements. Pixels, rems, and pulling from our [typography extends](https://github.com/edx/edx-platform/blob/master/lms%2Fstatic%2Fsass%2Felements%2F_typography.scss) are your best bet in the current platform.